modules/media/help/edit.md

```diff
@@ -0,0 +1,4 @@
+# Edit Media File
+
+This page allows users to modify information about an uploaded file.  Fields such as Date of Administration, Comments, and file visibility (hidden or visible) can be updated.  Note that fields such as PSCID, Visit Label, "Site", and "Instrument" cannot be modified on uploaded media files. 

```

Comment:
should "PSCID" and "Visit Label" have quotes too? (same with all the other places that columns are referenced in media.md)

Reply:
also "Date of Administration", "Comments", "file visibility"..